content/information-distribution/article.md

```diff
@@ -30,4 +30,48 @@ If the original news item contains media, these media **should** be uploaded to
 the node using [Matrix's content repository
 module](https://matrix.org/docs/spec/client_server/r0.4.0.html#id112).
 
-2️⃣: rest of the page
+## Matrix event `network.informo.article`
+
+This message event is meant to be sent by a source Matrix user. If the article
+is signed, it **must** be wrapped into a [Signed Matrix
+event](/information-distribution/signature/#signed-matrix-event).
+
+If the sender is not a source user, the article **should** be ignored. If the
+source registers itself afterwards, its previously sent articles **should**
+become visible.
+
+
+### Event data
+
+|      Parameter      |   Type   | Req. |                                                                            Description                                                                            |
+| ------------------- | -------- | :--: | ----------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `title`             | `string` |  x   | Article's headline                                                                                                                                                |
+| `href`              | `string` |      | Link to the article's original post (in case the article was sent to informo from an existing website)                                                            |
+| `short_description` | `string` |      | Short description or introduction to the article.                                                                                                                 |
+| `author`            | `string` |      | Full name of the article's author (when a single Informo source aggregates multiple writers).                                                                     |
+| `thumbnail`         | `string` |      | Preview image for the article. Must be a `mxc://` url.                                                                                                            |
```

Comment:
Might be worth adding a link to the mxc specs (which would be https://matrix.org/docs/spec/client_server/r0.4.0.html#id112)

Reply:
Also, please write "URL" in upper case since it's an acronym.